CMakeLists.txt

```diff
@@ -10830,13 +10831,6 @@ endif (gRPC_BUILD_TESTS)
 
 
 
-if (gRPC_INSTALL)
```

Comment:
What purpose was this supposed to serve and is this alright to remove for all environments?